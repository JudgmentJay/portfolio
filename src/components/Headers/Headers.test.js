import React from 'react'
import renderer from 'react-test-renderer'

import { Headers } from '../'

it('Renders correctly', () => {
	const tree = renderer
		.create(<Headers />)
		.toJSON()

	expect(tree).toMatchSnapshot()
})
