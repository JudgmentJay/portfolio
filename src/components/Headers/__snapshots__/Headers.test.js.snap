// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<div
  className="headers"
>
  <h1
    className="header"
  >
    Jay Posten
  </h1>
  <h2
    className="subheader"
  >
    Software Engineer
  </h2>
</div>
`;
